src/interface/src/styleguide/expander-section/expander-section.component.html

```diff
@@ -0,0 +1,15 @@
+<details [open]="isOpen">
```

Comment:
@lastminutediorama the expander icon its not the default arrow thingy.
Not sure if we can use this with custom icons.

Reply:
I ended up switching for just doing this manually. Didn't like how much I needed to override from the native details + summary elements in order for this to work with custom icons.

Reply:
got it.